README.md

```diff
@@ -1,16 +1,171 @@
-# Title
+# Nested Data Structures: Printing and Coalescing Data
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Display the cells in an `Array` of `Array`s
+* Traverse `Array` of `Array`s to produce a single value
+* Traverse `Array` of `Array`s to produce a new nested data structure
 
 ## Introduction
 
-## SWBAT 1
+When we started this module, we mentioned that we often use nested data
+structures as a "base" from which to do data processing. Arrays of arrays is
```

Comment:
This is a couple lessons too late, but it might be worth noting that it's also called "multidimensional arrays", because that seems to come up the most when you're trying to search around online for information.

Reply:
Will add to lesson prior.